### PR TITLE
 the missing lists

### PR DESCRIPTION
**Pull request by womullan**

### CODE REVIEW

File: communications/mail.rst [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#############
+Mailing lists
+#############
+Email is not the preferred mode of contact within DM but there are several project mailing list in use,
```
Comment (timj) [inline communications/mail.rst:4]:
```suggestion
Email is not the preferred mode of contact within DM but there are several project mailing lists in use.
```

File: communications/mail.rst [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+here is the full set of `Mailing lists`_.
```
Comment (timj) [inline communications/mail.rst:5]:
```suggestion
The full set can be found at `Mailing lists`_. 
```

File: communications/mail.rst [hunk 3]
```diff
@@ -0,0 +1,15 @@
+#############
+Mailing lists
+#############
+Email is not the preferred mode of contact within DM but there are several project mailing list in use,
+here is the full set of `Mailing lists`_. 
+These lists are mainly populated by the Contacts Database groups.
+
+The relevant lists for DM are :
```
Comment (timj) [inline communications/mail.rst:8]:
```suggestion
The relevant lists for DM are:
```

File: communications/mail.rst [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+- dm-cams: Data Management CAM's
```
Comment (timj) [inline communications/mail.rst:9]:
```suggestion
- `dm-cams <https://lists.lsst.org/mailman/listinfo/dm-cams>`_: Data Management CAM's
```

File: communications/mail.rst [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+- dm-sst : DM System Science Team
```
Comment (timj) [inline communications/mail.rst:10]:
```suggestion
- `dm-sst <https://lists.lsst.org/mailman/listinfo/dm-sst>`_: DM System Science Team
```

File: communications/mail.rst [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+- dm-staff: All DM construction staff
```
Comment (timj) [inline communications/mail.rst:11]:
```suggestion
- `dm-staff <https://lists.lsst.org/mailman/listinfo/dm-staff>`_: All DM construction staff 
```

File: communications/mail.rst [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+Email is not the preferred mode of contact within DM but there are several project mailing list in use,
+here is the full set of `Mailing lists`_. 
+These lists are mainly populated by the Contacts Database groups.
+
+The relevant lists for DM are :
+- dm-cams: Data Management CAM's
+- dm-sst : DM System Science Team
+- dm-staff: All DM construction staff 
+
+However these are seldom used and mentioned here for completeness.
```
Comment (timj) [inline communications/mail.rst:13]:
```suggestion
However, these are seldom used and mentioned here for completeness.
```

**Review by timj — APPROVED** [review]
Looks okay. With my suggestions I have no idea whether I got the rst link syntax correct.

**Comment by womullan** [comment]
you got it correct

**Pull request merged**